Sources/SwiftSyntaxBuilder/BuildableNodes.swift.gyb

```diff
@@ -114,6 +119,23 @@ public struct ${type.buildable()}: ${base_type.buildable()}, ${type.expressible_
   public func create${type.buildable_base_name()}() -> ${type.buildable()} {
     return self
   }
+% if has_with_trailing_comma_trait:
+%   trailing_comma_init_args = []
+%   for child in children:
+%     if child.name() == 'trailingComma':
+%       trailing_comma_init_args.append('%s: withComma ? .comma : nil' % (child.name()))
+%     else:
+%       trailing_comma_init_args.append('%s: %s' % (child.name(), child.name()))
+%     end
+%   end
```

Comment:
I think your indentation is wrong here. This should be indented two more spaces because it’s inside the `for` loop started in line 23. This also applies to line 138.